pkg/server/grpc/tracee.go

```diff
@@ -11,6 +13,30 @@ type TraceeService struct {
 	pb.UnimplementedTraceeServiceServer
 }
 
+func (s *TraceeService) GetEventDefinition(ctx context.Context, in *pb.GetEventDefinitionRequest) (*pb.GetEventDefinitionResponse, error) {
+	id, ok := events.Core.GetDefinitionIDByName(in.Name)
+	if !ok {
+		return nil, fmt.Errorf("event %s not found", in.Name)
+	}
```

Comment:
We also need to support the case where no name is given, which means we want to get all event definitions